.github/workflows/cd-pipeline.yaml

```diff
@@ -1,4 +1,4 @@
-name: 'CD Pipeline'
+name: '[CI/CD] CD Pipeline'
```

Comment:
I think the original intention of @FraPazGal was to differentiate CI pipeline vs CD pipelines.

Reply:
Right now they are appearing together casually because if we add a new workflow starting with `CX` where `X` is any letter between `I` and `D` it is going to be placed in the middle.

I think using this prefix style we can a proper order and the usability is better for Bitnami maintainers and external users.

Being said that, I used `[CI/CD]` but we can use any other string of course